backend/app/services/share/base_service.py

```
@@ -1009,6 +1011,174 @@ def add_member(
+
+        # Get or create share link once for all members
+        resource_type_variants = [self.resource_type.value]
+        if self.resource_type.value == "KnowledgeBase":
+            resource_type_variants.append("KNOWLEDGE_BASE")
+
+        share_link = (
+            db.query(ShareLink)
+            .filter(
+                ShareLink.resource_type.in_(resource_type_variants),
+                ShareLink.resource_id == resource_id,
+                ShareLink.is_active == True,
```

Ruff (0.15.6)

[error] Avoid equality comparisons to `True`; use `ShareLink.is_active:` for truth checks

Replace with `ShareLink.is_active`

(E712)